packages/@aws-cdk/aws-iotevents/lib/event.ts

```diff
@@ -1,18 +1,39 @@
 import { Expression } from './expression';
+import { State } from './state';
 
 /**
- * Specifies the actions to be performed when the condition evaluates to TRUE.
+ * The base interface for events.
  */
-export interface Event {
+interface IEventBase {
   /**
    * The name of the event.
    */
   readonly eventName: string;
+}
 
+/**
+ * Specifies the actions to be performed when the condition evaluates to TRUE.
+ */
+export interface IEvent extends IEventBase {
```

Comment:
I wouldn't introduce this new interface `IEventBase` (see my comments below on why).